<ui version="4.0" >
 <class>FileDialog</class>
 <widget class="QDialog" name="FileDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FileDialog</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="closeButton" >
   <property name="geometry" >
    <rect>
     <x>990</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>831</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton" >
   <property name="geometry" >
    <rect>
     <x>870</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save Files...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>860</x>
     <y>20</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Statistics</string>
   </property>
   <widget class="QWidget" name="gridLayout_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Number Of Events</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Number Of Files:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit_2" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Files size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lineEdit_5" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>860</x>
     <y>230</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title" >
    <string>Selected</string>
   </property>
   <widget class="QWidget" name="gridLayout" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Number Of Events</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit_3" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit_4" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Number Of Files:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lineEdit_6" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Files size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
